tests_recording/test_api.py

```diff
@@ -33,8 +32,8 @@ def setUp(self):
 
     def check_version_increase(self):
         # change specfile little bit to have there some change
-        specfile_location = os.path.join(self.lp.working_dir, "python-ogr.spec")
-        with open(specfile_location, "r") as myfile:
+        specfile_location = Path(self.lp_working_dir, "python-ogr.spec")
+        with open(str(specfile_location), "r") as myfile:
             filedata = myfile.read()
```

Comment:
This can be now simply:
```python
filedata = specfile_location.read_text()
```

Reply:
I like that move to `Path` more and more...